p4-14/v1.0.3/tex/p4.tex

```diff
@@ -658,7 +658,7 @@
 Only valid packet header fields may result in a match (when a value is
 specified for exact or ternary matches against the field), although a
 match operation may explicitly check if a header instance (or field)
-is valid. Only valid packet headers are considered for deparsing (see
+is valid (TBD: Should this be updated as asked elsewhere?). Only valid packet headers are considered for deparsing (see
```

Comment:
I'm in favor of amending this statement ("Only valid packet header fields may result in a match") for the following reasons:
- P4_14 offers a way to explicitly match of header validity
- this seems to be the direction p4_16 has chosen
- bmv2 does not check for validity in a match unless explicitly asked in the P4 program